tests/unit/test_pipeline.py

```diff
@@ -55,7 +55,10 @@ def test_no_container(test_dir, caplog, mocker):
         error_no=errno.EINVAL,
         expected_msg={
             "IMPLEMENTATION ERRORS": {
-                "pvs_like_python": [
+                "step_1_python_pandas": [
```

Comment:
Just curious if this error is like a FileNotFoundError or the error we same on Jenkins when you dont have access to slurm or the OsError no such device?

Reply:
Are you referring to `errno.EINVAL`? This is the standard errno value for "Invalid Argument" (errno 22 I think?)

https://docs.python.org/3/library/errno.html